packages/react-email/src/utils/get-emails.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'node:fs/promises';
+import { existsSync } from 'node:fs';
+import path from 'node:path';
+
+export const getEmails = async () => {
+  const emailsDirPath = path.join(
+    process.cwd(),
+    process.env.EMAILS_DIR_RELATIVE_PATH ?? 'emails',
+  );
+
+  if (!existsSync(emailsDirPath)) return [];
+
+  const emailPaths = await fs
+    .readdir(emailsDirPath, { withFileTypes: true })
```

Comment:
should we use `readdirSync` instead and remove the `async` from the function signature?